source/tyr/migrations/versions/35d86bbbe642_association_instances_equipments.py

```diff
@@ -0,0 +1,34 @@
+"""
+Association table between instances and equipments providers
+Remove unused column 'instances' in table 'equipments_provider'
+
+Revision ID: 35d86bbbe642
+Revises: 266658781c00
+Create Date: 2019-04-25 15:09:21.276526
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = '35d86bbbe642'
+down_revision = '266658781c00'
+
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects import postgresql
+
+
+def upgrade():
+    op.create_table(
+        'associate_instance_equipments',
+        sa.Column('instance_id', sa.Integer(), nullable=False),
+        sa.Column('equipments_id', sa.Text(), nullable=False),
+        sa.ForeignKeyConstraint(['equipments_id'], ['equipments_provider.id']),
+        sa.ForeignKeyConstraint(['instance_id'], ['instance.id']),
+        sa.PrimaryKeyConstraint('instance_id', 'equipments_id', name='instance_equipments_pk'),
+    )
+    op.drop_column(u'equipments_provider', 'instances')
```

Comment:
minor: next time try to do two migration as these change aren't really related

Reply:
OK. They are kind of related though, the column is dropped because the association table is now used